tests/attacks/test_threshold_attack.py

```diff
@@ -0,0 +1,170 @@
+# MIT License
+#
+# Copyright (C) IBM Corporation 2018
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the  rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""
+This module tests the Threshold Attack.
+
+| Threshold Attack Paper link:
+    https://arxiv.org/abs/1906.06026
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+import unittest
+
+import numpy as np
+
+from tests.utils import TestBase
+from tests.utils import get_classifier_tf, get_classifier_kr, get_classifier_pt
+
+from art.attacks import ThresholdAttack
+from art.utils import get_labels_np_array
+
+logger = logging.getLogger(__name__)
+
+
+class TestThresholdAttack(TestBase):
+    """
+    TODO: Write Comment
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        TODO: Write Comment
```

Comment:
Please update docstring.

Reply:
I could not find a sample docstring for the given function.